manifest.bioimage.io.yaml

```diff
@@ -40,3 +40,24 @@ model:
     links:
       - Ilastik
     download_url: https://github.com/subeeshvasu/hbp-DL-seg-codes/releases/download/0.1.2/2sUNetDA.model.zip
+
+dataset:
+  - id: stardist_dsb_training_data
+    authors:
+    - {name: Constantin Pape}
```

Comment:
This is something we need to be careful with as we don't want this to be the actual author field.
It would be much better to just use the maintainer field here.